application/templates/static_site/measure.html

```diff
@@ -119,60 +145,9 @@ <h2 class="govuk-heading-l">There is a new version of this page</h2>
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <div class="metadata">
-        <dl>
-          {% if measure_version.primary_data_source and measure_version.primary_data_source.publisher %}
-            <dt>Department:</dt>
-            <dd itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
-             <span itemprop="name">{{ measure_version.primary_data_source.publisher.name }}</span>
-            </dd>
-          {% endif %}
 
-          {%  if measure_version.first_published_date %}
-            <dt>Published:</dt>
-            <dd><time itemprop="datePublished" datetime="{{ measure_version.first_published_date | format_iso8601_date }}">{{ measure_version.first_published_date | format_friendly_date }}</time>
-            </dd>
-          {%  endif %}
+      <h2 class="govuk-heading-l" id="main-facts-figures">1. Main facts and figures</h2>
```

Comment:
Possibly just include the `and` in the `id` here, so that it reads more nicely (e.g. in the URL)? 100% optional. 🤷‍♂